tests/test_occ_powercap.robot

```diff
@@ -130,6 +130,7 @@ Get Chassis URI
 
 Host PowerOn
     [Documentation]     Keyword to poweron the host and wait till boot up
+    [Tags] host-power
```

Comment:
> Host PowerOn is a prereq keyword which I'm running as testsuite setup and needed for occ tests.

Right - but doesn't this make it a useful entry-point for disabling all the tests via tag exclusion?

> If host poweron is not supported on the qemu based platform then we should have a way to find out the current running platform and return from keyword if platform is qemu.
refer - http://robotframework.org/robotframework/latest/libraries/BuiltIn.html#Return%20From%20Keyword%20If

Given the QEMU environment is intended to (but doesn't currently) have all the functionality of the hardware, I'm not convinced distinguishing the platform is a useful approach. When the QEMU environment does reach feature parity, we can just not exclude the tag when invoking `pybot`.

If we are going to go down the path of detecting the platform through the keyword, what would you do to detect the environment? QEMU's `palmetto-bmc` machine boots the images generated for the hardware, and I'd be hesitant to put QEMU-specific bits into the production image for the sake of automated testing.

For what it's worth, as far as I can see the patch's approach is functional. Maybe it's not elegant, but I need more information on your suggested approach.